Kickstarter-iOS/Views/Cells/SettingsNotificationCell.swift

```diff
@@ -92,9 +91,12 @@ final class SettingsNotificationCell: UITableViewCell, NibLoading, ValueCell {
 
     self.emailNotificationsButton.rac.selected = viewModel.outputs.emailNotificationsEnabled
     self.emailNotificationsButton.rac.hidden = viewModel.outputs.emailNotificationButtonIsHidden
+    self.emailNotificationsButton.rac.accessibilityLabel =
+      viewModel.outputs.emailNotificationAccessibilityLabel
```

Comment:
Can we change the names of these 2 signals?

So instead of `emailNotificationAccessibilityLabel` do `emailNotificationsButtonAccessibilityLabel` and also instead of `pushNotificationAccessibilityLabel` do `pushNotificationsButtonAccessibilityLabel`.

This should follow the pattern of `component+property`.

Reply:
Because these buttons behave more like switches I was not sure that referring to it as a button . was accurate. Let me know what you think.

Reply:
I realize we're not really using them as buttons, but technically the component (`UIButton`) and the name of the component (`emailNotificationsButton`) refer to a button so we should probably stay consistent. The behaviour should be considered a side effect IMO